test/controllers/DashboardControllerSpec.scala

```diff
@@ -39,23 +67,76 @@ class DashboardControllerSpec extends FrontEndTestHelper {
     "render the judgment dashboard page with an authenticated judgment user" in {
       val controller = new DashboardController(
         getAuthorisedSecurityComponents,
-        getValidJudgmentUserKeycloakConfiguration)
-      val dashboardPage = controller.dashboard().apply(FakeRequest(GET, "/dashboard"))
+        getValidJudgmentUserKeycloakConfiguration,
+        consignmentService)
+      val dashboardPage = controller.dashboard().apply(FakeRequest(GET, "/dashboard").withCSRFToken)
       status(dashboardPage) mustBe OK
       contentType(dashboardPage) mustBe Some("text/html")
-      contentAsString(dashboardPage) must include ("Judgment Transfer Welcome")
+
+      contentAsString(dashboardPage) must include ("Welcome")
       contentAsString(dashboardPage) must include ("Welcome to the Transfer Digital Records service")
-      contentAsString(dashboardPage) must include ("Upload your judgment to start a new transfer")
-      contentAsString(dashboardPage) must include ("Placeholder page")
+      contentAsString(dashboardPage) must include ("Upload your records to start a new transfer")
```

Comment:
Think this wording should actually be different for the judgments, so might be able to test for this to ensure the correct version of the page is being displayed.

Reply:
I've changed it here. I've removed the check for the form element and just check for `Upload your judgment...`